docs/applications/simple-bank-account/src/main/java/com/scalar/application/bankaccount/service/AccountService.java

```diff
@@ -68,21 +68,21 @@
 
   private ResponseEntity<String> serve(ThrowableFunction f, JsonObject json) {
     try {
-      ContractExecutionResponse response = f.apply(json);
-
-      if (response.getStatus() == StatusCode.OK.get()) {
-        return ResponseEntity.ok(response.getResult());
-      } else {
-        return ResponseEntity.badRequest()
-            .body("status: " + response.getStatus() + ", message: " + response.getMessage());
-      }
+      ContractExecutionResult result = f.apply(json);
+
+      return ResponseEntity.ok(result.getResult().toString());
```

Comment:
It is probably not correct.
First, `result.getResult()` returns `Optional` so it shouldn't do `toString()` directly.
Second, is it OK to set null to `ResponseEntity.ok` ? If it is not, it should return `ResponseEntity.ok()` when it is empty.
https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/http/ResponseEntity.html#ok-T-

Reply:
Thank you for review comment.
Changes updated in d02b87aac798ec1dca93c22bc09f02a3f63d8161
```
return ResponseEntity
              .ok(result.getResult().isPresent() ? result.getResult().get().toString() : null);
```